.github/.codecov.yml

```diff
@@ -3,12 +3,12 @@
 
 codecov:
   notify:
-    require_ci_to_pass: no
+    require_ci_to_pass: yes
 
 coverage:
   precision: 2
   round: down
-  range: "70...100"
+  range: "50...75"
 
```

Comment:
Sorry am missing some context. Any reason why we reduced coverage range and what are the implications ?

Reply:
This is range is just setting the [color of the range](https://docs.codecov.io/docs/coverage-configuration), not changing the approval in any way. With this change, any file with at least 75% coverage would be marked as green.